metrics/thresholds.go

```diff
@@ -188,7 +188,12 @@ func (ts *Thresholds) Run(sink Sink, duration time.Duration) (bool, error) {
 		ts.sinked["min"] = sinkImpl.Min
 		ts.sinked["max"] = sinkImpl.Max
 		ts.sinked["avg"] = sinkImpl.Avg
-		ts.sinked["med"] = sinkImpl.Med
+
+		// Computing the trend's median value is equivalent
+		// to computing its 50th percentile value.
+		// As this can turn out to be a rather computationally
+		// expensive operation, we do this just in time.
```

Comment:
```suggestion
```

I don't feel like this comment adds much - if you are unaware that the 50th percentile is the median, this comment doesn't tell you why this is true anymore then the code. 

I also have problems with the "just in time" part as I would argue it would've been "just in time" if we did just before *actually* using it, not just before maybe needing it ;)